javascript/app-model-exposure/src/api-client/FetchApiInteractor.utils.ts

```diff
@@ -4,26 +4,82 @@
  */
 
 import { api } from '../resources/api.js'
-import type { NodeResponseStatus } from '../types/api/NodeResponseStatus.js'
+import { setStorageItem } from '../state/sessionStorage.js'
+import { NodeResponseStatus } from '../types/api/NodeResponseStatus.js'
 import type { StatusResponse } from '../types/api/StatusResponse.js'
 import type { ApiType } from './FetchApiInteractor.types.js'
 import { isProcessingStatus, wait } from './utils.js'
 
-export async function checkStatus(
+export async function checkStatus({
+	taskId,
+	type,
+	updateFn,
+	_updateId,
```

Comment:
It seems weird that you would have an arg here marked "private" with the underscore